src/components/Modal/ModalHeader.js

```diff
@@ -30,11 +30,12 @@ const ModalHeader = (props) => {
   };
 
   const isEmpty = !children && !tagline && !title;
+  const theme = prompt === true ? 'error' : prompt;
```

Comment:
`prompt && 'error'`

Reply:
I believe the short circuit wouldn't work the same if prompt is a string like 'info'. I did this to limit breaking changes and still allow prompt theming. Maybe it wasn't a good idea.

Reply:
Oh, I overlooked the change in PropType for prompt. Disregard the comment